### ISSUE TITLE
[Spoilers - all chapters] Translation inconsistencies

### ISSUE THREAD

**Issue by ItaloKnox** [07th-mod/onikakushi#181]
This discussion started on our Discord server, check the ``higurashi_dev`` channel if you want to see the whole thing.

Here's a summary of what happened:

* In Onikakushi, Irie is called ``manager``, but in the following chapters his title is changed to ``coach``
    * This is not necessarily wrong since both come from the same word in japanese, ``監督 (kantoku)``
* Ordinarily, that wouldn't be an issue, but Shion's also called ``manager`` in the following chapter
    * In this case, Shion is called ``マネージャー (maneejaa / manager)`` which directly refers to her as the manager of the baseball team
        * Worth to note that ``監督 (kantoku)`` and ``マネージャー (maneejaa)`` are ***not*** the same thing in japanese. The misleading only happens in english.
* A few people pointed out the inconsistency as being very detrimental to the story due to the misleading nature of Onikakushi's translation
    * Suggestion 1: change Shion's ``manager`` to ``captain``
        * I was against this change since they do not mean the same thing and can annoy people who are into sports manga/anime. A manager on a sports team is far from being the captain (or a coach).
    * Suggestion 2: change Irie's ``manager`` to ``director``
        * They essentially mean the same thing and the suggestion was well-received. This change would require a small revision in the script, but nothing that would take too much time. **Still up for discussion.**

Omake:

* [This line was also pointed out as inconsistent](https://cdn.discordapp.com/attachments/384427950478589972/556097619336757268/SPOILER_unknown.png)
    * I didn't review it yet so I can't say for sure, but if we trust what was said (the correct line is "You will meet the same fate as Tomitake"), it might look the same on a superficial level but I believe that's not what was intended to say in the original script

There is probably more stuff to look into Onikakushi, this should be just the start.

**Comment by JmTrad** [comment]
I share the same opinions as you, I had understood what they did here. It must be because of the influence of the old translation that people pointed this out. For me the current form is ok.

**Comment by tellowkrinkle** [comment]
BTW the last line in question is here in the script:

https://github.com/07th-mod/onikakushi/blob/4d6e83ec0a1353cd266eae0b0eea6a7a23cdbb52/Update/onik_015_02.txt#L1721-L1724

IMO using "You will meet the same fate" is weird given what's actually happening, I don't think anyone would actually say that.  At the same time, the current text (while I think it does have the meaning we want) is some seriously hard to parse English.

**Comment by vstax** [comment]
> change Irie's manager to director

won't it create more problems then? The trick behind "manager" word is that it made Keiichi confuse it with "manager of production site", that is dead foreman.

**Comment by enumag** [comment]
Talked with Italo privately about this, here are the results of the discussion:

One question was why we can't just use coach on Oni. Iirc it was because it's too specific and not mysterious/scary enough for the context. And Keiichi would not mistake him for the foreman.

As for director causing issues, I checked Oni and there is no usage of "manager" in Oni before this so the foreman was never referred to as such. In my opinion "director" could cause Keiichi to do the same mistake as well.

I believe we only need to fix this in Onikakushi because in other chapters Irie is usually referred to as "coach".

So with these in mind I'm changing "manager" to "director" in Onikakushi only.

Let me know if you think there is some counter-reason or anything to do in any other chapters. Thanks.

**Comment by enumag** [comment]
https://github.com/07th-mod/onikakushi/commit/270946caf41bd5ef8143bf690ee9b103300e9e18

**Comment by enumag** [comment]
No feedback so I'll assume it's ok. Closing.
